src/scripts/main.js

```diff
@@ -1,3 +1,187 @@
 'use strict';
 
-// write code here
+const mainTable = document.querySelector('table');
+const newForm = document.createElement('form');
+
+newForm.classList.add('new-employee-form');
+
+const submitButton = document.createElement('button');
+
+submitButton.innerText = 'Save to table';
+
+const countryForSelect = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+const allInputsForForm = [
+  createInput('text', 'Name'),
+  createInput('text', 'Position'),
+  createSelect(countryForSelect, 'Office'),
+  createInput('number', 'Age'),
+  createInput('number', 'Salary'),
+  submitButton,
+];
+
+newForm.append(...allInputsForForm);
+mainTable.after(newForm);
+
+function createInput(typeInput, nameInput) {
+  const newLabel = document.createElement('label');
+  const newInput = document.createElement('input');
+
+  newLabel.append(`${nameInput}:`);
+  newInput.name = nameInput.toLowerCase();
+  newInput.type = typeInput.toLowerCase();
+  newInput.dataset.qa = nameInput.toLowerCase();
+  newInput.required = 'true';
```

Comment:
The `newInput.required` property should be set to a boolean value `true` instead of the string `'true'`. This ensures proper validation behavior in the form.